docs/content/modules/tensor_mechanics/systems/Kernels/StressDivergenceRSphericalTensors.md

```diff
@@ -1,8 +1,24 @@
-<!-- MOOSE Object Documentation Stub: Remove this when content is added. -->
-#StressDivergenceRSphericalTensors
+#Stress Divergence RSpherical Tensors
+!description /Kernels/StressDivergenceRSphericalTensors
+
 !devel /Kernels/StressDivergenceRSphericalTensors float=right width=auto margin=20px padding=20px background-color=#F8F8F8
 
-!description /Kernels/StressDivergenceRSphericalTensors
+##Description
+This kernel solves the steady state stress divergence equation in polar coordinates on a 1D mesh.
+
+{!content/modules/tensor_mechanics/CommonDocumentation/StressDivergenceKernels.md!}
```

Comment:
I've changed the file name structure in my latest commit to `common_documents/supplementalStressDivergenceKernels.md` Does this help with the identification of the C++ object associated md files?